config.proto

```diff
@@ -109,13 +110,17 @@ enum PropagationFormat {
 
 // TraceReporterType represents the reporting format for trace data.
 enum TraceReporterType {
+
+    // Default to none
```

Comment:
```suggestion
    // Default to none. In such situation, tracer should act as noop.
```

As I said we need to clearly establish here the behaviour for agents cc @davexroth @pavolloffay

Reply:
Should the agents set a default type always? The agents create a default config always, at that time, they should set this to either ZIPKIN or OTLP. https://github.com/hypertrace/agent-config/pull/59/files#diff-3eaf2c85e69e9ea4804954dd98f3b0e3de1047da36714aef929a41efac2fbd54R56

Reply:
And a user should never give `UNSPECIFIED` as trace_reporter_type in config file or env vars.

Reply:
> // Default to none. In such situation, tracer should act as noop.

In this case the agent should use the default reporter - e.g. Zipkin or OTLP

Reply:
agreed.  when set to none, the agent should use it's default, so maybe change the comment to:
"Default to none.  Agent will use it's default reporting type"